documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,126 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add an Open Virtual Appliance (OVA) source provider by using the {ocp} web console.
+
+.Prerequisites
+
+* TBD
+endif::[]
+ifdef::ostack[]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {ocp} web console.
+
+[IMPORTANT]
+====
+Migration using {osp} source providers is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Adding a Red Hat {virt} source provider
+
+You can add a Red Hat {virt} source provider by using the {ocp} web console.
+
+.Prerequisites
+
+* TBD
+endif::[]
 
 .Procedure
 
 . In the {ocp} web console, click *Migration* -> *Providers for virtualization*.
 . Click *Create Provider*.
 ifdef::vmware[]
-. Select *VMware* from the *Provider type* list.
+. Click *VMware vSphere*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *vCenter host name or IP address*: vCenter host name or IP address - if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *vCenter user name*: vCenter user, for example, `user@vsphere.local`
-* *vCenter password*: vCenter user password
-* *VDDK init image*: VDDKInitImage path
+* *Provider resource name*: Name to display in the list of providers.
+* *URL*: Endpoint.
+* *VDDK init image*: VDDKInitImage path.If left empty, some functionality will not be available.
```

Comment:
@yaacov @ahadas Do we want a note about why it's a bad idea to leave it blank or do we want to just tell them to fill im the info? Is this a detail users might have trouble finding or are likely to leave blank for some other reason?

Reply:
@RichardHoch due to licensing issues we are not able to provider VDDK image with MTV and users are supposed to create the image themselves so it can be handy, in some scenarios, to skip it. we should highlight that it is strongly recommended to use VDDK in order to accelerate migrations

Reply:
Suggestion:
* *VDDK init image*: `VDDKInitImage` path. It is strongly recommended to create a VDDK init image to accelerate migrations.